subjects/atm-management-system/audit/README.md

```diff
@@ -6,73 +6,73 @@
+##### Referring to the user Alice, create an account but with account number `"3214"` and type of account `"fixed02"` with the rest of the information the same as the last account. Then select `"Check accounts"` and choose the account you just created.
 
-###### Did the application displayed the account information and the gain of $100.12 of interest on 10/10/2014?
+###### Did the application display the account information and the gain of $100.12 of interest on 10/10/2014?
 
-##### Resorting to the user Alice create again an account but with account number `"3212"` and type of account `"fixed03"` with the rest of the information as in the last account. Then select `"Check accounts"` and choose the account you just created.
+##### Referring to the user Alice, create an account but with account number `"3212"` and type of account `"fixed03"` with the rest of the information the same as the last account. Then select `"Check accounts"` and choose the account you just created.
 
-###### Did the application displayed the account information and the gain of $240.29 of interest on 10/10/2015?
+###### Did the application display the account information and the gain of $240.29 of interest on 10/10/2015?
 
-##### Resorting to the user Alice select the option `"Make transaction"`. Then choose the account with the id `"3212"`
+##### Referring to the user Alice, select the option `"Make transaction"`. Then choose the account with the id `"3212"`
 
 ###### Are you able to choose between withdrawing or depositing?
 
-##### Resorting to the user Alice select the option `"Make transaction"`, choose the account with the id `"3212"`. Then try to withdraw money.
+##### Referring to the user Alice select the option `"Make transaction"`, choose the account with the id `"3212"`. Then try to withdraw money.
 
 ###### Are you able to withdraw money?
 
 ###### And if so, was the withdraw updated in the file `"records.txt"`?
 
-###### Is it not possible to withdraw an amount superior to your available balance?
+###### Is it possible to withdraw an amount superior to your available balance?
 
 ##### Try to deposit money into the account `"3212"`.
 
 ###### Were you able to deposit money into this account?
 
 ###### And if so did it update the file `"records.txt"`?
 
-##### Resorting to the user Alice try to select the option `"Remove existing account"` and remove the accounts `"834213"`, `"320421"` and `"3214"`.
+##### Referring to the user Alice try to select the option `"Remove existing account"` and remove the accounts `"834213"`, `"320421"` and `"3214"`.
 
 ###### Can you confirm that those account were deleted, both in the application and file `"records.txt"`?
 
-##### Resorting to the user Alice select the option `"Remove existing account"` and try to remove and nonexisting account.
+##### Referring to the user Alice select the option `"Remove existing account"` and try to remove a account that doesn't exist.
```

Comment:
we try to avoid using the apostrophe to make the English simple and easy for every one, because not all the students have good English. So, you need to avoid using (don't, doesn't, ...) and instead use (do not, does not, ...)

##### Using the user Alice select the option `"Remove existing account"` and try to remove an account that does not exist.

Reply:
should not it be `remove **an** account that does not exist`